sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidInstantiationCheck.java

```diff
@@ -197,8 +197,9 @@ private static boolean isWildcardForbiddenImported(String importText,
         final String importTextWithoutAsterisk =
                 importText.substring(0, importText.length() - 1);
 
+        // -@cs[EqualsAvoidNull] need parenthesis around '+' otherwise PMD will complain
         return importText.endsWith("*")
-                && (importTextWithoutAsterisk + className).equals(forbiddenClassNameAndPath);
+                && forbiddenClassNameAndPath.equals(importTextWithoutAsterisk + className);
```

Comment:
I couldn't saitisfy PMD and Checkstyle on these types.
PMD wanted the new version, Checkstyle wanted the old version.